web-registry/src/components/molecules/ProjectPageMetadata/ProjectPageMetadata.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from 'react';
+import { Box, Collapse, Grid, useTheme } from '@mui/material';
+
+import { ExpandButton } from 'web-components/lib/components/buttons/ExpandButton';
+import { Title } from 'web-components/lib/components/typography';
+import { formatDate } from 'web-components/lib/utils/format';
+
+import { ProjectMetadataLDUnion } from '../../../generated/json-ld';
+import { ArrowLink } from '../../atoms/MetadataArrowLink';
+import { MetaDetail } from '../MetaDetail';
+
+export interface MetadataProps {
+  metadata?: Partial<ProjectMetadataLDUnion>;
+  projectId?: string;
+}
+
+const ProjectPageMetadata: React.FC<React.PropsWithChildren<MetadataProps>> = ({
```

Comment:
renamed from `ProjectMetadata` to `ProjectPageMetadata` to differentiate from create flow component. 
Consolidated `.VCS` and `.CFC` components into this single component.